lib/publiccloud/utils.pm

```diff
@@ -188,4 +202,12 @@ sub is_container_host() {
     return is_public_cloud && get_var('FLAVOR') =~ 'CHOST';
 }
 
+sub define_secret_variable {
+    my ($var_name, $var_value) = @_;
+    script_run("set -a");
+    script_run("read -sp \"enter value: \" $var_name", 0);
+    type_password($var_value . "\n");
```

Comment:
This is smart 😆 👍

Reply:
surprisingly credits goes to MS documentation :) I was just smart enough to find this and read :)